frontend/packages/console-shared/src/components/formik-fields/multi-column-field/MultiColumnField.scss

```diff
@@ -1,23 +1,24 @@
 .odc-multi-column-field {
   &__row {
-    display: flex;
-    flex-direction: row;
     position: relative;
     margin-right: var(--pf-global--spacer--lg);
     margin-bottom: var(--pf-global--spacer--lg);
     max-height: var(--pf-global--spacer--xl);
   }
 
   &__col {
-    flex: 1 0 0;
     margin-right: var(--pf-global--spacer--md);
   }
 
   &__col--button {
-    cursor: pointer;
     line-height: var(--pf-global--spacer--xl);
     position: absolute;
     right: -30px;
+    top: 50%;
+    transform: translateY(-50%);
+    button:not(:disabled) {
+      cursor: pointer;
+    }
```

Comment:
Why is this needed when we are using PF Button which already has a pointer cursor?

Reply:
🤔 good point... I just moved the cursor without testing if it was needed.